lib/Slackbot_worker.js

```diff
@@ -1,17 +1,21 @@
 var Ws = require('ws');
+var _ = require('lodash');
 var request = require('request');
 var slackWebApi = require(__dirname + '/Slack_web_api.js');
 var HttpsProxyAgent = require('https-proxy-agent');
 
 
 module.exports = function(botkit, config) {
+    config = config || {};
     var bot = {
         botkit: botkit,
-        config: config || {},
+        config: _.clone(config),
         utterances: botkit.utterances,
-        api: slackWebApi(botkit, config || {})
+        api: slackWebApi(botkit, _.clone(config))
     };
 
+    var botApi = slackWebApi(botkit, bot.config);
```

Comment:
I'm not really quite sure why you didn't use `bot.api` but preferred a different variable exactly as the same one and use `botApi`